modules/ipi-install-configuring-raid-for-worker-node.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-raid-for-worker-node_{context}"]
+= Configuring RAID for worker node
+
+The following procedure configures RAID (Redundant Array of Independent Disks) for the worker node during the installation process.
+
+[NOTE]
+====
+. Only servers with bmc type `irmc` are supported. Other types of servers are currently not supported.
+====
+
+.Procedure
+. Create manifests.
+. Modify the BMH (Bare Metal Host) file corresponding to the worker:
++
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+----
++
+[NOTE]
+====
+Because servers with BMC (Baseboard Management Controller) type `irmc` does not support software RAID, the following RAID configuration uses hardware RAID as an example.
+====
++
+.. If you added specific RAID configuration to the spec, this will cause the worker node to delete the original RAID configuration
+in the `preparing` phase and perform a specified configuration on the RAID. For example:
++
+----
+spec:
+  raid:
+    hardwareRAIDVolumes:
+	- level: "0"
+	  name: "sda"
+	  numberOfPhysicalDisks: 1
+	  rotational: true
+	  sizeGibibytes: 0
+----
+<1> `level` is a required field, and the others are optional fields. See the
+link:https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#raid[BMH documentation]
+for details.
++
+.. if you added an empty RAID configuration to the spec, this empty configuration causes the worker node to delete the original RAID configuration during the `preparing` phase, but will not perform a new configuration. For example:
++
+----
+spec:
+  raid:
+    hardwareRAIDVolumes: []
+----
++
+[NOTE]
+====
+. The previous two situations require that the server has a RAID controller, otherwise an error will occur during the `preparing` phase.
```

Comment:
Remove the . in the note, which renders this as a numbered step. Also consider placing this note at the beginning of the procedure or, even better, make "The server must have a RAID controller" a prerequisite before the procedure.